provisioning_client/samples/readme.md

```diff
@@ -0,0 +1,19 @@
+# Samples for Azure IoT Provisioning SDK
+
+## List of samples
+
+- custom_hsm_examples
+
+  - This example shows how to create a custom HSM to retrieve the secruity information needed for the provisioning SDK to communication the the DPS Service.
+
+- iothub_client_sample_hsm
+
+  - This sample show how to use the IoThub Device Client with the device authentication system without using provisioning.
```

Comment:
why is this in the provisioning_client folder if it doesn't use provisioning?

Reply:
Good point.

---
In reply to: [254838897](https://github.com/Azure/azure-iot-sdk-c/pull/854#discussion_r254838897) [](ancestors = 254838897)